applications/zigbee_weather_station/CMakeLists.txt

```diff
@@ -6,18 +6,10 @@
 
 cmake_minimum_required(VERSION 3.20.0)
 
-
-if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}")
```

Comment:
Could you explain, why this check is considered as an issue?

Reply:
Zephyr already checks that APPLICATION_CONFIG_DIR exists and will error out if it does not